test/e2e/healthcheck_run_test.go

```diff
@@ -54,6 +54,28 @@ var _ = Describe("Podman healthcheck run", func() {
 		Expect(hc).Should(Exit(125))
 	})
 
+	It("podman run healthcheck and logs should contain healthcheck output", func() {
+		session := podmanTest.Podman([]string{"run", "--name", "test-logs", "-dt", "--health-interval", "1s", "--health-cmd", "'echo working'", "busybox", "sleep", "3600"})
```

Comment:
```suggestion
		session := podmanTest.Podman([]string{"run", "--name", "test-logs", "-dt", "--health-interval", "1s", "--health-cmd", "echo working", "busybox", "sleep", "3600"})
```
AS commented before you have to remove the extra quotes, right now the healcheck is failing with `/bin/sh: echo working: not found`

Reply:
Fixed actually i fixed it last time as well. Earlier i was not able to figure out why CI was failing so i just added it back for experimentation